src/main/java/com/programmers/commandline/domain/voucher/repository/impl/MemoryVoucherRepository.java

```diff
@@ -0,0 +1,41 @@
+package com.programmers.commandline.domain.voucher.repository.impl;
+
+import com.programmers.commandline.domain.voucher.entity.Voucher;
+import com.programmers.commandline.domain.voucher.repository.VoucherRepository;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+@Profile({"local"})
+public class MemoryVoucherRepository implements VoucherRepository {
+
+    private List<Voucher> memory = new ArrayList<>();
+
+    @Override
+    public String save(Voucher voucher) {
+         memory.add(voucher);
+        return voucher.getId();
```

Comment:
```suggestion
        memory.add(voucher);
        return voucher.getId();
```
save시 자동 코드정렬을 이용해보세용 :)

Reply:
감사합니다. 조언 덕분에 커맨드 + s 할때마다 코드가 이뻐지고 있습니다.🍔